server/controllers/finance/journal/index.js

```diff
@@ -251,6 +253,9 @@ function editTransaction(req, res, next) {
     .then((currentTransaction) => {
       const posted = currentTransaction[0].posted;
 
+      // bind the current transaction under edit as "oldTransaction"
+      oldTransaction = currentTransaction;
```

Comment:
I'll update references to make use of `oldTransaction` and `_oldTransaction` for when it just needs to be exposed to a larger scope.